server/scripts/createSuperUser.ts

```diff
@@ -1,16 +1,33 @@
 import { PrismaClient } from '@prisma/client'
+import axios from 'axios'
+import FormData from 'form-data'
 
 const prismaClient = new PrismaClient()
 
-const { SUPERUSER_EMAIL, SUPERUSER_PASSWORD } = process.env
+const {
+  SUPERUSER_EMAIL,
+  SUPERUSER_PASSWORD,
+  IDENTITY_PROVIDER_URL,
+} = process.env
 
 const main = async () => {
   if (!SUPERUSER_PASSWORD) throw Error('superuser password is required')
+  const user = {
```

Comment:
maybe we should add a check whether the user already exist in pyrog and skip everything if that's the case, what do you think ?